content/en/docs/acknowledgements/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Acknowledgements
+spelling: cSpell:ignore otel
+aliases: [/acknowledgements]
+weight: 200
+---
+
+We would like to acknowledge the following sources for some of the content on this site:
+
+1. [APM is Dying and That's Okay - Lightstep](https://lightstep.com/blog/apm-is-dying-and-thats-okay)
+2. [Tracing Concepts - github/cartermp](https://gist.github.com/cartermp/d4202c531bf73e369fb945e53a1a0045)
+3. [What is OpenTelemetry - Dynatrace](https://www.dynatrace.com/news/blog/what-is-opentelemetry-2/)
+4. [Understanding OpenTracing, OpenCensus, and OpenMetrics - BMC](https://www.bmc.com/blogs/opentracing-opencensus-openmetrics/)
+5. [Ask Miss O11y: Baggage in OTel - Honeycomb](https://www.honeycomb.io/blog/ask-miss-o11y-opentelemetry-baggage/)
```

Comment:
I managed to paraphrase most of the content cited here, and for the ones that are a direct ref, I've gotten permission to use the content.

Reply:
Yep confirming that all Honeycomb stuff of this manner (esp. the Ask Miss O11y posts) are okay to pull in and then just attribute via link here.